CMakeLists.txt

```diff
@@ -85,6 +85,19 @@ else ()
 endif ()
 
 ###########################################################################
+#       CCache configuration
+###########################################################################
+
+find_program(CCACHE_FOUND ccache)
+if(CCACHE_FOUND)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
+    if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Qunused-arguments -fcolor-diagnostics")
```

Comment:
There is no need to silence unused argument. There is only one such function: unimplemented binary Memcached protocol `handle_received_data`.
-fcolor-diagnostics may affect unix pipeline where build output used with `grep` or `sed`.
It's better to use such flags during the build time:
```CXXFLAGS=-fcolor-diagnostics make```